### PR TITLE
Update fix-issues-with-printers-scanners-and-lob-applications-that-se…

### PR DESCRIPTION
**Pull request by arindamthokder**
…nd-email-using-off.md

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 1]
```diff
@@ -126,6 +126,12 @@ There are a few things you should check:
 
 1. Enable Client SMTP submission on the licensed mailbox being used:
 
+   Run the following Powershell command-
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:129]:
```suggestion
   Run the following PowerShell command:
```

File: Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md [hunk 2]
```diff
@@ -126,6 +126,12 @@ There are a few things you should check:
...
+
+      Set-CASMailbox -Identity sean@contoso.com -SmtpClientAuthenticationDisabled $false
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mail-flow-best-practices/fix-issues-with-printers-scanners-and-lob-applications-that-send-email-using-off.md:131]:
```suggestion
    ```PowerShell
   Set-CASMailbox -Identity sean@contoso.com -SmtpClientAuthenticationDisabled $false
   ```
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @arindamthokder, kindly commit the suggested changes. Many thanks.

**Comment by arindamthokder** [comment]
> Copy review - @arindamthokder, kindly commit the suggested changes. Many thanks.

done 👍

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**